benchmarks/lambda/Snakefile

```diff
@@ -0,0 +1,56 @@
+rule lambda_generate:
+	input:
+                script="benchmarks/lambda/analysis/gen_lambda_decay.cxx",
+	params:
+		NEVENTS_GEN=100000,
+	output:
+		GEN_FILE="lambda_decay_{P}GeV.hepmc"
+	shell:
+		"""
+root -l -b -q '{input.script}({params.NEVENTS_GEN},0,"{output.GEN_FILE}",{wildcards.P},{wildcards.P})'
+"""
+
+rule lambda_simulate:
+	input:
+		GEN_FILE="lambda_decay_{P}GeV.hepmc"
+	params:
+		NEVENTS_SIM=3000,
+		PHYSICS_LIST="FTFP_BERT"
+	output:
+		SIM_FILE="{DETECTOR_CONFIG}_sim_lambda_dec_{P}GeV.edm4hep.root"
+	shell:
+		"""
+# Running simulation
+npsim \
+   --compactFile $DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml \
+   --numberOfEvents {params.NEVENTS_SIM} \
+   --physicsList {params.PHYSICS_LIST} \
+   --inputFiles {input.GEN_FILE} \
+   --outputFile {output.SIM_FILE}
+"""
+
+rule lambda_recon:
+        input:
+                SIM_FILE="{DETECTOR_CONFIG}_sim_lambda_dec_{P}GeV.edm4hep.root"
+        params:
+                NEVENTS_REC=3000,
+        output:
+                REC_FILE="{DETECTOR_CONFIG}_rec_lambda_dec_{P}GeV.edm4hep.root"
+        shell:
+                """
+eicrecon -Pplugins=ZDC {input.SIM_FILE} -Ppodio:output_file={output.REC_FILE} -Pdd4hep:xml_files=$DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml -Ppodio:output_include_collections=MCParticles,HcalFarForwardZDCClusters,HcalFarForwardZDCRecHits,HcalFarForwardZDCSubcellHits  -Pjana:nevents={params.NEVENTS_REC}
```

Comment:
`-Pplugins=ZDC` is broken in recent JANA2 https://chat.epic-eic.org/main/pl/mmyspc5uhfrp7khu1tz9p8kr8o
Let's just not use it
```suggestion
eicrecon {input.SIM_FILE} -Ppodio:output_file={output.REC_FILE} -Pdd4hep:xml_files=$DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml -Ppodio:output_include_collections=MCParticles,HcalFarForwardZDCClusters,HcalFarForwardZDCRecHits,HcalFarForwardZDCSubcellHits  -Pjana:nevents={params.NEVENTS_REC}
```